### DISCUSSION TITLE
ps1 file encoding

### DISCUSSION THREAD

**Discussion by alexaxb** [AdhocAdam/smletsexchangeconnector#234] [Q&A] [answered]
Are ps-files executed with UTF-8 encoding as default in US, or is that a machine specific setting?
When I'm updating the connector, I have to convert it from UTF-8 to ANSI before I can get PS to interpret it correctly...

**Comment by AdhocAdam** [comment]
Not sure/I have never once thought to question this until (to your credit) some of the recent points you've raised @alexaxb 😄

Is there an error or a weird character? Is it the _entire_ script or just parts of it?

**Reply by alexaxb** [reply]
Haha..glad to help 😄
It looks there are som special space-characters scattered around in the script, and if the script is not viewed as UTF8, the character is just jibberish..
![image](https://user-images.githubusercontent.com/14937505/102612533-27b04b00-4131-11eb-8af2-8e96debfd334.png)

**Reply by AdhocAdam** [reply]
Looks like I was started new sub-threads here as opposed to direct replies. Not sure if that impacted notifications to you? In any case - take a look at #235 when you get a chance. While the diff makes it seem as though just spaces were removed they in fact appear to be the characters you refer to in the screenshot above.

**Comment by AdhocAdam** [comment]
Your screenshot gave me an idea. I opened in ISE instead of VSCode and I see what you're talking about now. This is strange to say the least but something that seems easily rectified. I'll check this out in depth this weekend and try to at least get a branch open around file encoding.

**Comment by AdhocAdam** [comment, marked as answer]
Take a look at https://github.com/AdhocAdam/smletsexchangeconnector/pull/235 when you get a chance. While the diff makes it seem as though just spaces were removed they in fact appear to be the characters you refer to in the screenshot above.

**Reply by alexaxb** [reply]
Yea, it looks good now!

**Reply by AdhocAdam** [reply]
Excellent. Thank you for the heads up and confirmation sir!
